sdk/core/core-tracing/vitest.config.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 
 import { defineConfig } from "vitest/config";
+import { resolve } from "path";
```

Comment:
Make sure to use `node:` prefixes for anything from Node specific

Reply:
Fixed, thanks!